README.md

```diff
@@ -168,6 +168,15 @@ config :ueberauth, Ueberauth,
   ]
 ```
 
+## Customizing JSON Serializer
+
+Your json encoder/decoder can be configured.  Defaults to Poison in other ueberauth libraries such as [ueberauth_google](https://github.com/ueberauth/ueberauth_google) and [ueberauth_twitter](https://github.com/ueberauth/ueberauth_twitter).  If you only have [Jason](https://github.com/michalmuskala/jason) installed, then it is a good idea to take advantage of this configuration property.
```

Comment:
@ueberauth/core do we need this note or should we push out releases with all strategies to leverage this configuration option?  I also think we should default to Jason in the new release.

Reply:
@doomspork yep, we need to release a patch version for this config, and a minor version for Ueberauth.

Also, I agree on putting `Jason` as default. `  Application.get_env(:ueberauth, :json_library, Jason)`